socorro/external/es/crashstorage.py

```diff
@@ -211,6 +211,77 @@ def save_raw_and_processed(self, raw_crash, dumps, processed_crash,
 
 
 #==============================================================================
+class ESCrashStorageRestrictedDump(ESCrashStorage):
```

Comment:
Changed to ``ESCrashStorageRedactedJsonDump``. But note that the JSON dump is simply a JSON version of the dump.